tests/water.py

```diff
@@ -0,0 +1,20 @@
+import os
+import os.path as osp
+import numpy as np
+
+
+def test_dervs(coordinates, charges, order):
```

Comment:
No need to pass coordinates and charges: the values were obtained with a specific geometry and for one single molecule.
```suggestion
def reference_h2o_derivative(order):
```